rdrf/report/TrrfGraphQLView.py

```diff
@@ -0,0 +1,25 @@
+import logging
+
+from graphene_django.views import GraphQLView
+from graphql import GraphQLError
+from django.utils.translation import ugettext_lazy as _
+
+logger = logging.getLogger(__name__)
+
+
+class PublicGraphQLError(GraphQLError):
+    pass
+
+
+class TrrfGraphQLView(GraphQLView):
+
+    @staticmethod
+    def format_error(error):
+        super_class = super(TrrfGraphQLView, TrrfGraphQLView)
```

Comment:
Seems a bit unusual at first glance, but I think that's the best way of using `super` on a static method. Otherwise, you could just call `GraphQLView.format_error(...)` directly, but using python's _fun_ inheritance model is preferable

Reply:
Yeah. I did a bit of reading on the best way to call super on a static method. Seems a bit odd, but thought better to do it this way than call it directly in case the super class is swapped out for something else (even though that's pretty unlikely).